web/src/components/lineage/components/node/Node.tsx

```diff
@@ -43,25 +46,52 @@ const Node: React.FC<NodeProps> = ({ node, selectedNode, setSelectedNode }) => {
     return '/'
   }
 
+  // configure the tool tip
+  const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
+    <Tooltip {...props} classes={{ popper: className }} />
+  ))(({ theme }) => ({
+    [`& .${tooltipClasses.tooltip}`]: {
+      backgroundColor: theme.palette.background.default,
+      color: theme.palette.common.white,
+      maxWidth: 600,
+      fontSize: theme.typography.pxToRem(14),
+      border: '1px solid ' + theme.palette.common.white,
+    },
+  }))
+
+  // return the object namespace/name
+  const addToToolTip = (inputData: any) => {
+    const inputString = String(inputData.label)
+    const desc = String(inputData.data.description)
+    const namespace = String(inputData.data.namespace)
+    const objectName = String(inputData.data.name)
+    return <React.Fragment>
+      <Typography color="inherit">{inputString.split(':')[0]}</Typography>
+      <b>{"Namespace: "}</b>{namespace}<br></br>
+      <b>{"Name: "}</b>{objectName}<br></br>
+      <b>{"Description: "}</b>{desc === 'null' ? "No Description" : desc}
```

Comment:
I think this should use the `children` prop in order for the component to be generally configurable.